apps/opik-frontend/src/components/ui/select.tsx

```diff
@@ -109,23 +109,31 @@ const SelectLabel = React.forwardRef<
 ));
 SelectLabel.displayName = SelectPrimitive.Label.displayName;
 
+interface SelectItemProps
+  extends React.ComponentPropsWithoutRef<typeof SelectPrimitive.Item> {
+  withoutCheck?: boolean;
+}
+
 const SelectItem = React.forwardRef<
   React.ElementRef<typeof SelectPrimitive.Item>,
-  React.ComponentPropsWithoutRef<typeof SelectPrimitive.Item>
->(({ className, children, ...props }, ref) => (
+  SelectItemProps
+>(({ className, children, withoutCheck = false, ...props }, ref) => (
```

Comment:
@andriidudar take a look at it in case you have some strong opinion about it

Reply:
Good implementation, no comments from my side.